ios/engine/KMEI/KeymanEngine/Classes/Constants.swift

```diff
@@ -124,3 +124,22 @@ public enum FileExtensions {
   public static let openTypeFont = "otf"
   public static let configurationProfile = "mobileconfig"
 }
+
+public enum KeymanError: Error, LocalizedError {
+  case unknown
+  // file that is missing, is the file critical?
+  case missingFile(String, Bool)
+
+  var localizedDescription: String {
+    switch self {
+      case .unknown:
+        return engineBundle.localizedString(forKey: "error-unknown", value: nil, table: nil)
```

Comment:
Can the pre-localized error description get sent to Sentry?

Reply:
Probably should convert a lot of the `log.error` logging into Sentry reports, but that's a separate target for another day, not for this PR.